src/abort.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
oh god we have to remove that eslint rule. i hate default export :)

Reply:
Done!